src/components/ui/Badge/badge.scss

```diff
@@ -27,6 +36,24 @@
   background-color: var(--bg-info);
 }
 
+[data-store-badge-variant="interactive"] {
+  > span {
+    padding-right: 0;
+  }
+
+  [data-store-icon-button] {
+    width: var(--tap-min-size);
+    height: auto;
+    color: var(--color-neutral-7);
+    border-top-right-radius: var(--border-radius-pill);
+    border-bottom-right-radius: var(--border-radius-pill);
+
+    &:hover {
+      background-color: var(--color-secondary-1);
```

Comment:
Will this be the same color for differently colored badges? I'm assuming it's always a darker tone than the variant's background color. But then we'd have to specify the different colors for each variant I guess. Maybe there's a way to keep it DRY using a color function, but not sure yet.

Reply:
nice catch @filipewl, after you raised this point I talked to Flip and his advice is to use black as the hover color with low opacity, I'll apply that